Nitaicandra/CENPROJECT
# Login UI

### user story
As a user, I want to easily log-in/log-out in the application so that I can see personalized information relevant to me

### summary of changes
- created a UI for the login page (starting page atm)
- added authentication functionality to the frontend

IMPORTANT: must create a .env file in the client folder. Check discord for instructions

### how to test
Test UI Functionality
1.  rebuild the containers and run them
2. Create an account using Postman if your database is empty / never created an account before
3. go to http://localhost:5173/ and try to login. Check if the page changes.
4. also check if all css stylings and images are working

Test token functionality
1. Right click anywhere in the page and go to inspect > console
2.  Type` window.localStorage`. You should see your token and some info about your account. 

### logout and create an account
Logout functionality is not implemented yet. You can manually log out by typing ` window.localStorage.clear()` in the console

It's not possible to create an account from the UI yet. Please use postman in the meantime.
